.github/_CI-PUBLISHING.md

```
@@ -0,0 +1,87 @@
+1. Create a granular access token on npmjs.com with publish access
+2. Add it as a repository secret named `NPM_TOKEN`
+3. Modify the workflow to use:
+   ```yaml
+   env:
+     NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+   ```
+
+## Version Scheme
+
+CI versions follow the pattern:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)